src/gui/editors/TimeLineWidget.cpp

```diff
@@ -60,22 +63,18 @@ TimeLineWidget::TimeLineWidget(const int xoff, const int yoff, const float ppb,
 	m_autoScroll( AutoScrollState::Enabled ),
 	m_changedPosition( true ),
 	m_xOffset( xoff ),
-	m_posMarkerX( 0 ),
 	m_ppb( ppb ),
 	m_snapSize( 1.0 ),
 	m_pos( pos ),
 	m_timeline{&timeline},
 	m_begin( begin ),
 	m_mode( mode ),
 	m_hint( nullptr ),
-	m_action( Action::NoAction ),
-	m_moveXOff( 0 )
+	m_action( Action::NoAction )
```

Comment:
```suggestion
	m_action(Action::NoAction)
```

Reply:
I considered making this change, but if I were to "fix" this line, I'd rather do so by moving the initialisation to the member declaration in the header, at which point that changes the line above (removing the comma), and then that needs fixing, and so on...

I can go ahead with this if you'd like, though.

Reply:
I'd be fine with removing the above line and initializing in the header.